samples/cpp/benchmark_app/README.md

```diff
@@ -1,66 +1,107 @@
 # Benchmark C++ Tool {#openvino_inference_engine_samples_benchmark_app_README}
 
-This topic demonstrates how to use the Benchmark C++ Tool to estimate deep learning inference performance on supported devices.
-Performance can be measured for two inference modes: latency- and throughput-oriented.
+This page demonstrates how to use the Benchmark C++ Tool to estimate deep learning inference performance on supported devices.
 
-> **NOTE**: This topic describes usage of C++ implementation of the Benchmark Tool. For the Python* implementation, refer to [Benchmark Python* Tool](../../../tools/benchmark_tool/README.md).
+> **NOTE**: This page describes usage of the C++ implementation of the Benchmark Tool. For the Python implementation, refer to the [Benchmark Python Tool](../../../tools/benchmark_tool/README.md) page. The Python version is recommended for benchmarking models that will be used in Python applications, and the C++ version is recommended for benchmarking models that will be used in C++ applications. Both tools have a similar command interface and backend.
+		
 
+## Basic Usage
+To use the C++ benchmark_app, you must first build it following the [Build the Sample Applications](../../../docs/OV_Runtime_UG/Samples_Overview.md#build-the-sample-applications) instructions and then set up paths and environment variables by following the [Get Ready for Running the Sample Applications](../../../docs/OV_Runtime_UG/Samples_Overview.md#get-ready-for-running-the-sample-applications) instructions. Navigate to the directory where the benchmark_app C++ sample binary was built.
 
-## How It Works
+> **NOTE**: If you installed OpenVINO Runtime using PyPI or Anaconda Cloud, only the [Benchmark Python Tool](../../../tools/benchmark_tool/README.md) is available, and you should follow the usage instructions on that page instead.
 
-Upon start-up, the application reads command-line parameters and loads a network and inputs (images/binary files) to the specified device.
+The benchmarking application works with OpenVINO models in IR format (`model.xml` and `model.bin`) or ONNX format (`model.onnx`). You can use it with your own models or pre-trained [public](@ref omz_models_group_public) or [Intel](@ref omz_models_group_intel) models from the Open Model Zoo, which can be downloaded using the [Model Downloader](@ref omz_tools_downloader). Before running the tool, make sure the model is converted to OpenVINO format using the [Model Optimizer](../../../docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md) tool or is in ONNX format.
```

Comment:
My suggestion would be to remove the OMZ part from this sentence because in this context this may add some confusion. As a general rule, for OMZ models you should use Model Converter, not MO. And intel models are already in IR format. But if we add all that it is too much information. From my perspective the part from "You can use it with your own models..." could be deleted, but if we want to keep it, can it be just a link to OMZ with if needed a note that public models must be converted with omz_converter (linking to the documentation page about that)?